### ISSUE TITLE
Need to have proper validations according to the fields in whole product 

### ISSUE THREAD

**Issue by rashmi0301** [Fleet-Hawks-Inc/fh-cloud-app#958]
1. need to have proper validations in whole product according to the respective fields . 
if we add invalid data the validation should appear .
if we submit the blank form then validation should appear

applicable for fleet , routing and address book , onboard, login  , models

**Comment by rashmi0301** [comment]
@nameetjambla is working on it ..

**Comment by rashmi0301** [comment]
@nameetjambla kindly mention the names of pages in which validations are already done .. so that i will start testing on those pages ..

**Comment by nameetjambla** [comment]
@rashmi0301  From fleet module Assets, fuel, vehicles, issues,  are completed (reminder) is also done yesterday but some new fields are added today so I need to put validation once again you can check reminder validation but don't consider it complete at a moment. 
From routing and dispatch you can check validation on the add order page for a moment for add-trip validation is done but i need to check on my end so I will let you know once it up on the dev link

**Comment by rashmi0301** [comment]
checked asset 
issues :- 
1. asset name validation is not correct 
2. VIN - this is the alphanumeric and i had added alphanumeric 18 characters but still validation is showing .
3. Notes -- no validation apply 

checked Fuel 
1. fuel card field is pending 
2. no character limit in reference 


checked issue :- 
1. no limit in odometer 
2. i had  added .- with issue name still validation message is appear 
3. no validation in description

**Comment by rashmi0301** [comment]
checked validations on order page 

1. additional contact name --- it should only alphabets small , capital , space .
2. reference field should be alphanumeric 
3. pickup and delivery instruction -- need to have validation 
4. commodity , temperature  -- need to have validation

**Comment by rashmi0301** [comment]
I am done the validations with Address book . 

@nameetjambla @sadaf510  is working on onboard , fleet  , routing

**Comment by robinhawks** [comment]
@rashmi0301 @nameetjambla How much time do we need to finish this guys ???

**Comment by sadaf510** [comment]
I have finished my part validations on routing and dispatch @robinhawks

**Comment by nameetjambla** [comment]
@rashmi0301  @robinhawks  done from my side once my updated PR is merged you will check.

**Comment by rashmi0301** [comment]
Address book validations are done and tested 
Routing Dispatch  validations are done and tested except cross border 
Fleet validations are tested but having issues in inventory + driver + vehicle + geofence

**Comment by nameetjambla** [comment]
@rashmi0301  you can check the validations of the entire fleet if you still face any problems do let me know.

**Comment by rashmi0301** [comment]
DONE
